changelog.d/6332.bugfix

```diff
@@ -0,0 +1 @@
+Fix caching devices for remote users when using workers.
```

Comment:
this is a bit cryptic. Perhaps you can explain what the user-visible symptoms were?